vnext/Shared/HermesSamplingProfiler.cpp

```diff
@@ -4,25 +4,42 @@
 #include "pch.h"
 
 #include <chrono>
+#include <future>
 
 #include <hermes/hermes.h>
-#include <winrt/Windows.Storage.h>
 #include "Unicode.h"
 
+#ifdef WINRT
+#include <winrt/Windows.Storage.h>
+#else
+#include <ShlObj.h>
+#include <Shlwapi.h>
+#endif
+
 #include "HermesSamplingProfiler.h"
 
 namespace Microsoft::ReactNative {
 
 namespace {
 std::future<std::string> getTraceFilePath() noexcept {
+  std::ostringstream os;
+#ifdef WINRT
```

Comment:
thanks for addressing the win32 case : ) 
I think this only needs a little tweak, I'm not sure where the WINRT #define comes from but I think this needs to just be a simple check:
```cpp
uint32_t length{0};
if (GetCurrentPackageFullName(&length, nullptr) != APPMODEL_ERROR_NO_PACKAGE) {
  // ... do UWP or MSIX stuff
} else {
  // do unpackaged stuff
}
```